File: GJSBackend/WebService/Models/WebRequestModel.cs
// *******************************************************************
// 業務名称　: 互助防疫システム
// 機能概要　: モデル定義
//
// 作成日　　: 2024.07.17
// 作成者　　: AIPlus
// 変更履歴　:
// *******************************************************************

namespace Jbd.Gjs.WebService
{
    public class WebRequestModel
    {
        //[FromBody]

        public string SERVICE_NAME { get; set; }
        public string METHOD_NAME { get; set; }
        public CmRequestJson BIZ_REQUEST { get; set; }

    }

    public class CmRequestJson
    {
        public string DATA { get; set; }       //データ
    }
}

File: backend/JbdGjsWebService/Models/WebRequestModel.vb
' *******************************************************************
' 業務名称　: 互助事業システム
' 機能概要　: モデル定義
'
' 作成日　　: 2024.07.17
' 作成者　　: 宋 峰
' 変更履歴　:
' *******************************************************************

Namespace JBD.GJS.WebService
    Public Class WebRequestModel
        '[FromBody]
        Public Property SERVICE_NAME As String
        Public Property METHOD_NAME As String
        Public Property BIZ_REQUEST As CmRequestJson

    End Class

    Public Class CmRequestJson
        Public Property DATA As String       'データ
    End Class
End Namespace
